<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoAI_Precarga Modelo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="paginador">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CONFIGURAR EL MODELO</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Seleccionar la imagen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QComboBox" name="listImagen">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2" colspan="3">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Banda Rojo (Red)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="bandR">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Banda Verde (Green)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="bandG">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Banda Azul (Blue)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="bandB">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3" rowspan="5" colspan="2">
        <widget class="QLabel" name="icon_portada">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>120</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>fondoimagen.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MODELO SAM-------------------------------</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nombre del Modelo</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="nmodelo">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>vit_h</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vit_l</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vit_b</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Ruta al archivo del modelo</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" rowspan="2">
        <widget class="QPushButton" name="brmodelo">
         <property name="text">
          <string>Ubique el archivo</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLabel" name="ruta_file">
         <property name="text">
          <string>................................</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Procesamiento</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QComboBox" name="device">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>CPU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GPU si es posible</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>260</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="bejecutar">
         <property name="text">
          <string>EJECUTAR</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QPushButton" name="bcancelar">
         <property name="text">
          <string>CERRAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="opBasicas">
       <property name="geometry">
        <rect>
         <x>395</x>
         <y>220</y>
         <width>121</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>    RETORNAR</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/opAvanzadas.png</normaloff>icons/opAvanzadas.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
